corehq/ex-submodules/pillowtop/utils.py

```diff
@@ -136,15 +136,13 @@ def get_pillow_json(pillow_config):
     else:
         time_since_last = ''
         hours_since_last = None
-    try:
-        db_seq = pillow.get_change_feed().get_latest_change_id()
```

Comment:
I think you can remove `get_latest_change_id` now too. Pretty sure this was the only real usage.